src/main/connect/connectionManager.ts

```diff
@@ -53,6 +57,13 @@ export class ConnectionManager implements ExtensionComponent {
         return Promise.resolve(summaryResponse.artifacts);
     }
 
+    public async getGoCenterModules(componentDetails: ComponentDetails[]): Promise<IComponentMetadata[]> {
```

Comment:
Can we make this a static?

Reply:
No. You should use it exactly the same as `getComponents()`.